nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -166,6 +166,19 @@ let
       # Create a directory for exchanging data with the VM.
       mkdir -p "$TMPDIR/xchg"
 
+      ${lib.optionalString cfg.useHostCerts
+      ''
+        mkdir -p "$TMPDIR/certs"
+
+        NIX_SSL_CERT_FILE=''${NIX_SSL_CERT_FILE:-/etc/ssl/certs/ca-certificates.crt}
+
+        if [ -e "$NIX_SSL_CERT_FILE" ]; then
+          cp -L "$NIX_SSL_CERT_FILE" "$TMPDIR"/certs/ca-certificates.crt
```

Comment:
Is copying required (or better) than symlinking?

Reply:
From my testing, I believe copying is required here as the symlink gets resolved within the VM not on the host side